<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="general">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="periodSpinner">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>85</width>
       <height>22</height>
      </rect>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.300000000000000</double>
     </property>
     <property name="maximum">
      <double>3600.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="periodLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Card display time</string>
     </property>
    </widget>
    <widget class="QLabel" name="rowsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of rows</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rowsSpinner">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="colsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of colums</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="colsSpinner">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QTableView" name="cellTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>351</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="bgdLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Background color</string>
     </property>
    </widget>
    <widget class="ColorButton" name="colorButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="cardsets">
    <attribute name="title">
     <string>Card sets</string>
    </attribute>
    <widget class="QTableView" name="csetTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>361</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="csetAddBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="csetDelBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="csetRoot">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="csetRootLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Data root</string>
     </property>
    </widget>
    <widget class="QToolButton" name="csetRootButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>32</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>util/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
